### ISSUE TITLE
Password protection

### ISSUE THREAD

**Issue by rednaw** [rednaw/exhibitionDB#50]
We want users to authenticate (enter password) before accessing the data. There is no standard solution for this for static sites like those hosted on GitHub Pages.

A possible solution is to split the site in a public and private repository:
- All static HTML, CSS and Javascript go in the public repository
- All data (*.mer and *.sqlite3 files) goes in the private repository

The public repository serves the site and contains a login form, users login with their Github credentials after which the site uses those credentials to fetch the *.sqlite files from the private repo using the GitHub API.

Customer onboarding is done as follows:
- User creates a Github account if they not have any
- We grant that account access to the private repo

Basically this outsources the whole user administration infrastructure to GitHub

A description of this setup can be found at https://rmannibucau.metawerx.net/github-pages-authentication.html
